### PR TITLE
Update balances ethereum erc721 latest

### PR DESCRIPTION
**Pull request by denz-e**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

Migrating `balances_ethereum_erc721_latest` and `balances_ethereum_erc721_noncompliant` and updating the logic to rely on `nft_transfers` instead

### CODE REVIEW

**Comment by denz-e** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by denz-e** [comment]
recheck

File: models/balances/ethereum/erc721/balances_ethereum_erc721_schema.yml [hunk 1]
```diff
@@ -79,11 +79,12 @@ models:
...
     description: >
-        ERC721 NFT contract addresses that allow multiple minting(from 0x000..000) of the same tokenId.
-        Depends on erc721_ethereum_transfers.
+        ERC721 NFT contract address and tokenId pairs with multiple owners.
+        Depends on nft_transfers.
     columns:
-      - *token_address
\ No newline at end of file
+      - *token_address
+      - *tokenId
```
Comment (0xRobin) [inline models/balances/ethereum/erc721/balances_ethereum_erc721_schema.yml:90]: same comment here.
I think we revert this to only having the address and not the id.

**Review by 0xRobin — COMMENTED** [review]
looks good overall, one small comment on the non-compliant columns:

File: models/balances/ethereum/erc721/balances_ethereum_erc721_noncompliant.sql [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+            tokenId
+        FROM erc721_transfers
+        WHERE recency_index = 1
+        GROUP BY blockchain, token_address, tokenId
+        HAVING COUNT(wallet_address) > 1
+    )
 
-select distinct token_address as token_address FROM multiple_owners
\ No newline at end of file
+    SELECT * FROM multiple_owners
```
[the remark this answers is not in the document]
Comment (denz-e) [reply models/balances/ethereum/erc721/balances_ethereum_erc721_noncompliant.sql:31]: makes sense

File: models/balances/ethereum/erc721/balances_ethereum_erc721_schema.yml (hunk 1 above)
Comment (denz-e) [reply models/balances/ethereum/erc721/balances_ethereum_erc721_schema.yml]: got it

**Comment by denz-e** [comment]
thanks @0xRobin for the review - should be good to go now

**Review by 0xRobin — APPROVED** [review]
:heavy_check_mark: looks good.

**Comment by jeff-dude** [comment]
hi @denz-e -- i'm hesitant to proceed here, as we are actively working on a solution to balances outside of spellbook, which will then be considered a source that can be read into spellbook just like raw/decoded tables. please see this issue:
https://github.com/duneanalytics/spellbook/issues/4523#issuecomment-1827582865

are you okay if we close this PR, then wait for those balances to go live? we are putting an emphasis on getting balances on the platform and doing it as accurate and efficient as possible.

**Comment by denz-e** [comment]
Sounds good to me!

**Pull request closed without merging**